README.md

```diff
@@ -10,7 +10,12 @@
 
 # @videojs/vhs-utils
 
-Objects and functions shared throughtout @videojs/http-streaming code
+vhs-utils serves two purposes:
+
+1. It extracts objects and functions shared throughout @videojs/http-streaming code to save on package size. See [the original @videojs/http-streaming PR](https://github.com/videojs/http-streaming/pull/637) for details.
+2. It exports functions useful to plugin authors to mimic some logic VHS uses internally.
```

Comment:
Should this be updated as I thought we weren't doing 2 anymore?

Reply:
Yeah, I should re-phrase it. It still can be used to help plugin authors, but to reuse the functionality rather than mimicking internal logic.